love/src/Constants.js

```diff
@@ -3,6 +3,7 @@ export const MAX_CCW_FOLLOWING_ERROR = 2.0;
 export const AUTHLIST_REQUEST_PENDING = 'Pending';
 export const AUTHLIST_REQUEST_ACCEPTED = 'Authorized';
 export const AUTHLIST_REQUEST_DENIED = 'Denied';
+export const LOG_LEVELS = { debug: 10, info: 20, warning: 30, error: 40 };
```

Comment:
Why is this mapping defined like this?

Reply:
Good question! I've always had the doubt about when to use `Constants.js` or `Config.js` but I decided to set this variable here, at `Constants.js`, because of the nature of it... I would use this file to set all variables that should be always the same and for configurations the ones that could change. The difference is really thin, but I think, for example, that style mappings are ok to go at `Config.js`, as those variables are more likely to change, thus they should be considered as configurations. I hope I explained myself jeje, we can discuss about this anyways

Reply:
Ok, I understand. And what is the reason for this correspondence between the states and the values they have? For example, that "warning" has the value 30.

Reply:
Oh, I see. Thanks Sebastian!